test/gurka/test_indoor.cc

```diff
@@ -117,4 +133,15 @@ TEST_F(Indoor, EdgeInfo) {
   EXPECT_EQ(get_level_ref("A", "B"), "Parking");
   EXPECT_EQ(get_level_ref("B", "C"), "");
   EXPECT_EQ(get_level_ref("C", "F"), "Lobby");
+}
+
+TEST_F(Indoor, ElevatorPenalty) {
+  // first route should take the elevator node
+  auto result = gurka::do_action(valhalla::Options::route, map, {"E", "J"}, "pedestrian");
+  gurka::assert::raw::expect_path(result, {"EF", "FG", "GH", "HI", "IJ"});
+
+  // second route should take the stairs because we gave the elevator a huge penalty
+  result = gurka::do_action(valhalla::Options::route, map, {"E", "J"}, "pedestrian",
+                            {{"/costing_options/pedestrian/elevator_penalty", "3600"}});
+  gurka::assert::raw::expect_path(result, {"EF", "CF", "Cx", "xy", "yJ"});
```

Comment:
this is pretty straightforward. first we take a route that only has a 5 second penalty to take the elevator, which is obviously cheaper than turning and then takign the steps with their penalty. then we jack up the elevator penalty so that the steps become more appealing